app/integration_test/ui/views/history/history_test.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:app/main.dart' as app;
+import 'package:app/ui/widgets/atr_preview.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+void main() {
+  // Not in setUpAll() due to: https://github.com/flutter/flutter/issues/72063
+  // This will likely be fixed soon! Check again if you're adding tests here
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  Future<void> navigateToHistoryScreen(WidgetTester tester) async {
+    await tester.tap(find.widgetWithText(RaisedButton, "Skip sign in"));
+    await tester.pumpAndSettle();
+    await tester.tap(find.text("Travel History"));
+    await tester.pumpAndSettle();
+  }
+
+  // Some of these are system tests. They will likely be moved from history to system_test dir in the future.
+  group('History screen', () {
+    testWidgets(
+        "successfully retrieve one ATR from remote for specific Transporter",
+        (WidgetTester tester) async {
+      // Ensure app is in good state before testing
+      app.main();
```

Comment:
FYI: Running multiple integration tests in one file gets weird because of `getIt` being static for every test. Either we should have each system test in it's own file, or I continue to see if I can't find how to run them all in one.

Doing `setUp() { app.main(); }` does not destroy the previous run instance.